### DISCUSSION TITLE
Consider add library version for official documentation site.

### DISCUSSION THREAD

**Discussion by mrdulin** [reduxjs/redux-toolkit#4369] [General]
What is the library version corresponding to the documentation? Always the latest version?

Like this:

![image](https://github.com/reduxjs/redux-toolkit/assets/17866683/2becb201-78c3-46b9-b2f6-a2d501c44722)

**Comment by EskiMojo14** [comment]
yes, it's synced to master so generally will be whatever the latest version is

**Reply by mrdulin** [reply]
Thanks. Good to know.
